src/Propel/Generator/Behavior/NestedSet/NestedSetBehaviorObjectBuilderModifier.php

```diff
@@ -281,9 +281,10 @@ protected function addProcessNestedSetQueries(string &$script): void
 /**
  * Execute queries that were saved to be run inside the save transaction
  *
- * @param  ConnectionInterface \$con Connection to use.
+ * @param ConnectionInterface \$con Connection to use.
+ * @return void
```

Comment:
other classes have an empty line before return param

Reply:
We dont enforce CS just yet on those files.